tasks/HandMeThat.tex

```diff
@@ -0,0 +1,60 @@
+\section{Hand Me That [Party Host]}
+A guest at the party speaks English, but with only a limited vocabulary. The robot will assist them in obtaining things that they gesture for.
+
+%\subsection{Focus}
+%Joint attention is a well-studied and important task in Human-Robot Interaction. The goal of this task is to really challenge the teams to perform a hard HRI task.
+
+\subsection{Main Goal}
+Five collections of items are set out. The referee will choose a random item from each collection to gesture to or describe to the robot. The robot must tell the referee its name, point to, or touch it. The robot can prompt the user with requests like \textit{point to it} or \textit{describe it to me}.
+
+\noindent\textbf{Reward:} 2500pts\\
+
+
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Setup
+%
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsection{Setup}
+\begin{enumerate}
+	\item \textbf{Object Collections:} The test area should collections of objects on various surfaces (2-5). Difficulty of understanding the point gesture will be regulated by how far apart the objects in each collection are ($100cm - 1m$). Difficulty of describing the object is difficult to quantify, and should be considered random.
+
+	\item \textbf{Random Object Cards:} The referee will carry a set of cards with random objects, arranged by difficulty. The job of the referee is to get the robot to say the name of, or touch, the object that they gesture to or describe, at the robot's discretion.
+
+\end{enumerate}
+
+
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Setup
+%
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsection{Procedure}
+\begin{enumerate}
+	\item \textbf{Pick an object} The robot will ask the referee what they want. The referee will walk near to the object that they want, and gesture to it. The robot can ask clarifying questions to help identify the object. The referee will not say the name of the object, or its color, but anything else is fair game.
```

Comment:
Just wondering. Why specifically not color?

Reply:
It would make the problem too easy. 

Here's the algorithm if color is allowed:

Is it the red one? The black one? The green one?

Eliminating color tests that the robot understands the gesture or the language. If you leave it, this round becomes a free 2500 points.

Reply:
Hm, a problem I see is this:
A robot recognizes that there is an apple and a towel on the table but the team did not bother to do anything with gesture recognition. Now they could just guess which potentially results in less points so that is fine but instead they could also ask "Do you mean the fruit?". This is still just guessing but they will get full points every time. Maybe we should deduct points for clarifying questions similarly to wrong guesses?

Reply:
Yeah, @kyordhel made a related comment and I incorporated the change. Pushing in a moment.